mscr-ui/src/modules/crosswalk-editor/mappings-accordion/index.tsx

```diff
@@ -113,12 +113,12 @@ function Row(props: {
                       <StyledButton
                         className="px-3 py-0"
                         style={{textTransform: 'none'}}
-                        title={t('mappings-accordion.select-linked-nodes')}
+                        title={props.showAttributeNames ? t('mappings-accordion.select-linked-nodes') : returnFullPath(mapping.id)}
```

Comment:
Would it be helpful if the full path was visible on hover in both cases, so even if you have selected to view just node titles, you could still check the path by hovering?

Reply:
Previously this was hard-coded to use label by properties values. I changed properties values to use id instead. Practically there is only one mode used at the moment, but in theory properties could be used to choose what to show.

Reply:
Yes, the toggle for the other mode should be taken into use, see my first comment on this page. I think we can ditch the hard-coded label, because the full path will be useful in both modes.